### DISCUSSION TITLE
Hacking

### DISCUSSION THREAD

**Discussion by Lunatic226** [WalletConnect/walletconnect-monorepo#771] [Developer Support]
Somebody use fake Wallet connect to hack users...i pas a victim..is There a solution?

**Comment by chadyj** [comment]
Sorry to hear that you were hacked.

For what to do, take a look at https://walletconnect.com/faq#safety

**Comment by Lunatic226** [comment]
Thanks

Le jeu. 10 févr. 2022 à 13:40, Chad Jackson ***@***.***> a
écrit :
